packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/types/FlutterDataStoreFailureMessage.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_datastore.types
+
+enum class FlutterDataStoreFailureMessage {
+    CASTING {
+        override fun toString(): String {
+            return "ERROR_CASTING_INPUT_IN_PLATFORM_CODE"
+        }
+    },
+    MALFORMED {
+        override fun toString(): String {
+            return "AMPLIFY_REQUEST_MALFORMED"
```

Comment:
Can we make it more clear that Configure, Query, or another future method has the request malformed?
Maybe have specific error messages for malformed requests similar to API call failure messages you have below, or append the method name to the generic message when throwing the error above.

Reply:
Makes sense. Adding specific QUERY_MALFORMED and CONFIGURE_MALFORMED enum constants.